### PR TITLE
GJDifficultySprite bindings

### PR DESCRIPTION
**Pull request by AdyaGMD**

### CODE REVIEW

File: bindings/2.200/GeometryDash-extras.bro [hunk 1]
```diff
@@ -4185,16 +4185,16 @@ class GJCommentListLayer : cocos2d::CCLayerColor {
...
+	bool init(int, GJDifficultyName) = win 0x11f80;
 	~GJDifficultySprite();
 
-	TodoReturn getDifficultyFrame(int, GJDifficultyName);
+	TodoReturn getDifficultyFrame(int, GJDifficultyName) = win 0x212010;
 
-	TodoReturn updateFeatureState(GJFeatureState);
-	TodoReturn updateDifficultyFrame(int, GJDifficultyName);
-	TodoReturn updateFeatureStateFromLevel(GJGameLevel*);
+	TodoReturn updateFeatureState(GJFeatureState) win 0x2122c0;
```
Comment (Cvolton) [inline bindings/2.200/GeometryDash-extras.bro:4195]: where =

Comment (matcool) [reply bindings/2.200/GeometryDash-extras.bro:4195]: i agree

**Comment by xhayper** [comment]
where's the =

File: bindings/2.200/GeometryDash-extras.bro (hunk 1 above)
Comment (xhayper) [reply bindings/2.200/GeometryDash-extras.bro:4195]: where's the `=` again

**Comment by Cvolton** [comment]
and thus ends the saga of moving equal signs around

File: bindings/2.200/GeometryDash-extras.bro [hunk 2]
```diff
@@ -4185,16 +4185,16 @@ class GJCommentListLayer : cocos2d::CCLayerColor {
...
 
-	TodoReturn getDifficultyFrame(int, GJDifficultyName);
+	TodoReturn getDifficultyFrame(int, GJDifficultyName) = win 0x212010;
 
-	TodoReturn updateFeatureState(GJFeatureState);
-	TodoReturn updateDifficultyFrame(int, GJDifficultyName);
-	TodoReturn updateFeatureStateFromLevel(GJGameLevel*);
+	TodoReturn updateFeatureState(GJFeatureState) = win 0x2122c0;
+	TodoReturn updateDifficultyFrame(int, GJDifficultyName) = win 0x212100;
+	TodoReturn updateFeatureStateFromLevel(GJGameLevel*) = win Ox212240;
```
Comment (matcool) [inline bindings/2.200/GeometryDash-extras.bro:4197]: this should be a 0, not an O

**Pull request merged**